src/lib/Behat/BusinessContext/ContentViewContext.php

```diff
@@ -14,11 +18,56 @@ class ContentViewContext extends BusinessContext
 {
     /**
      * @Given I start creating a new Landing Page :name
+     *
+     * @param string $name
      */
     public function startCreatingNewLandingPage(string $name): void
     {
         $contentStructurePage = PageObjectFactory::createPage($this->utilityContext, ContentStructurePage::PAGE_NAME);
-        $contentStructurePage->createLandingPage($name, 'Test Desc');
+        $updatePage = $contentStructurePage->startCreatingContent('Landing page');
+
+        $updatePage->updateForm->fillFIeldWithValue('Title', $name);
+        $updatePage->updateForm->fillFIeldWithValue('Description', $name);
+    }
+
+    /**
+     * @Given I start creating a new Article :name
+     *
+     * @param string $name
+     */
+    public function startCreatingArticle($name): void
+    {
+        $contentStructurePage = PageObjectFactory::createPage($this->utilityContext, ContentStructurePage::PAGE_NAME);
+        $updatePage = $contentStructurePage->startCreatingContent('Article');
+
+        $updatePage->updateForm->fillFIeldWithValue('Title', $name);
+        $updatePage->updateForm->fillRichtextWithValue('Test desc');
+    }
+
+    /**
+     * @Given I start editing the content in :language language
+     */
```

Comment:
We should decide also if we comment parameters or not. We have both cases in our code.

Reply:
Ok, what's your opinion? I actually feel like they're not needed most of the time (and we should add them only when a description is really needed).

Reply:
I think the same, so maybe we just remove it where parameter meanings are trivial.